src/seeThru.js

```diff
@@ -249,22 +249,25 @@
 		, requestAnimationFrame = window.requestAnimationFrame || getRequestAnimationFrame()
 		, cancelAnimationFrame = window.cancelAnimationFrame || getCancelAnimationFrame()
 		, drawFrame = function(recurse){
+			var image, alphaData, i, len, currentTime = video.currentTime;
+			
+			if (lastFrameTime !== currentTime) {
+				lastFrameTime = currentTime;

```

Comment:
It'd be as confusing as lastFrameTime to me 😆 `lastDrawnFrameTime` is what I'd expect to read.